xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-war/src/main/webapp/resources/js/xwiki/usersandgroups/usersandgroups.js

```diff
@@ -142,13 +147,13 @@ window.MSCheckbox = Class.create({
 
       var action = "";
       var nxtst = (self.state + 1) % self.nrstates;
-
+      require(['xwiki-l10n!users-and-groups-translation-keys'], function(l10n) {
```

Comment:
The content of the function is missing an indent.

Reply:
Fixed in https://github.com/xwiki/xwiki-platform/pull/2991/commits/56ef94abdb1120ce8e41dae39afb907b564791fe :+1: